io/disk/softwareraid.py.data/Readme

```diff
@@ -1,11 +1,17 @@
 mdadm - manage MD devices aka Linux Software RAID 
 
-Values to be passed in ymal file:
+Values to be passed in yaml file:
 
 * Name of the devices on which this scripts should be run.
 
 * Raid levels to be created
 
+* Setup - True if raid needs to be created
+
+* Cleanup - True if raid needs to be deleted
+
+* Run_tests - True if mdadm tests needs to be run on created raid
+
```

Comment:
confusing...U create raid only from mdadm...can u explain, what u mean

Reply:
If setup and cleanup is set to False, and run_tests is set to True, then only the mdadm tests are run on it.

Reply:
can u rephrase it properly

Reply:
I am not sure what part you did not understand here.
So, it is better if you suggest the text.

Reply:
```Run_tests - True if mdadm tests needs to be run on created raid``` 
This confusing, you create raid using mdadm...and what mdadm tests you run on raid created with mdadm

Reply:
fail, remove, add device